website/sidebars.js

```diff
@@ -597,7 +597,7 @@ const sidebars = {
         {
           type: 'html',
           value:
-            '<a class="menu__link menu__list-item" href="/run-arbitrum-node/run-local-dev-node">Run a Stylus dev node<span class="other-section-icon">↑</span></a>',
+            '<a class="menu__link menu__list-item" href="/run-arbitrum-node/run-nitro-dev-node">Run a Stylus dev node<span class="other-section-icon">↑</span></a>',
```

Comment:
I vote for removing this menu item entirely?

Also, stylus `Quickstart (Rust)` references `nitro testnode`, would be good to just replace the nitro testnode section with a link to `run-nitro-dev-node`?

Reply:
As for removing it entirely from the sidebar, I still lean toward keeping it. I think some developers new to the ecosystem might want to experiment with Stylus, and this section of our docs is likely where they’ll go to start building. Keeping it visible ensures they can easily find it. What do you think?


(I’ll update the quick start)